src/main/java/step2/domain/Rank.java

```diff
@@ -0,0 +1,16 @@
+package step2.domain;
+
+public enum Rank {
+    POSTION_4(5000, 3),
+    POSTION_3(50000, 4),
```

Comment:
```suggestion
    POSTION_3(50_000, 4),
```
언더 스코어를 활용하면 가독성을 좀 더 올릴 수 있어요!